federation/pkg/dnsprovider/providers/google/clouddns/clouddns.go

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// clouddns is the implementation of pkg/dnsprovider interface for Google Cloud DNS
+package clouddns
+
+import (
+	"io"
+
+	"github.com/golang/glog"
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/dns/v1"
```

Comment:
Make this a named import

```
dns "google.golang.org/api/dns/v1"
```

Not doing this is bad for readability, even worse for grepping. I started scrambling for `dns`  when I saw it here https://github.com/kubernetes/kubernetes/pull/25389/files#diff-9e1208e1d901d951628091e4e80c06dcR103 because I did not see it defined anywhere before.